modules/classicon.lua

```diff
@@ -53,7 +53,7 @@ function ClassIcon:OnEnable()
 	end
 	
 	self:InsertTestDebuff(8122, 8, "Magic") -- Psychic Scream
-	self:InsertTestDebuff(19503, 4, nil) -- Scatter Shot

```

Comment:
this doesnt exist in cata, just remove the line

Reply:
gone